dev-setup.Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:focal
```

Comment:
This file is almost the same as the production and testing files, with the exception of not doing `yarn install` or `yarn build`  , because of `node_modules` not persisting if `yarn install` is performed in the Dockerfile, for reasons laid out in more detail here: https://stackoverflow.com/questions/30043872/docker-compose-node-modules-not-present-in-a-volume-after-npm-install-succeeds/32785014#32785014